File: ViewController.m
#import "ViewController.h"

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    UILabel *label = [[UILabel alloc] initWithFrame:CGRectMake(50, 50, 200, 50)];
    label.text = @"Hello, Objective-C!";
    [self.view addSubview:label];
}

@end

File: ViewController.swift
import UIKit

class ViewController: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
        
        let label = UILabel()
        label.text = "Hello, iOS!"
        label.frame = CGRect(x: 50, y: 50, width: 200, height: 50)
        self.view.addSubview(label)
    }
}
